extensions/sql-database-projects/src/test/projectController.test.ts

```diff
@@ -210,6 +212,41 @@ describe('ProjectsController', function (): void {
 				}
 			});
```

Comment:
Can you add a test for when a user already has a tasks.json file (for other extensions)?

Reply:
Certainly, I can include that scenario. To clarify, we are adding the .vscode/tasks.json file only for newly created projects. If a project already contains a tasks.json file but lacks a sqlproj-build task definition, Visual Studio Code will not be able to execute the Run Build Task command (Ctrl+Shift+B) by default.